src/EnergyPlus/HeatingCoils.cc

```diff
@@ -2067,6 +2065,11 @@ namespace HeatingCoils {
             }
         }
 
+        if (state.dataHeatingCoils->CoilIsSuppHeater) {
+            state.dataHVACGlobal->SuppHeatingCoilPower = heatingCoil.ElecUseLoad;
+        } else {
+            state.dataHVACGlobal->ElecHeatingCoilPower = heatingCoil.ElecUseLoad;
+        }
```

Comment:
This is already in ReportHeatingCoil so is not needed in the calc routines. It looks to me as if you only needed to delete the reference to ElecHeatingCoilPower, and other code, as you have done.

    void ReportHeatingCoil(EnergyPlusData &state, int const CoilNum, bool const coilIsSuppHeater)
    {

        if (coilIsSuppHeater) {
            state.dataHVACGlobal->SuppHeatingCoilPower = heatingCoil.ElecUseLoad;
        } else {
            state.dataHVACGlobal->ElecHeatingCoilPower = heatingCoil.ElecUseLoad;
        }

Reply:
That was my initial thought but when I looked at the code Unitary System calls the `CalcMultiStageElectricHeatingCoil` function directly in several places and it won't hit the `ReportHeatingCoil` function.

Reply:
Yes, that is to cut down on code execution during iterations. But as a final call, after the PLR is known, the parent should call the sim function for all coils and fan, which will call Report. See if that is happening and if not then there is the problem.

Reply:
Stepped through the code and confirmed that the report varaibles update code in `ReportHeatingCoil` is sufficient. Also ran example files and and the unit test for additional verification. I am going to remove the redundant code.

Reply:
Looks like this got resolved. :+1: